dashboard/test/models/user_test.rb

```diff
@@ -2803,6 +2807,47 @@ def track_progress(user_id, script_level, result, pairings: nil)
     assert @admin.authorized_teacher?
   end
 
+  test "authorized instructor" do
+    # normal teacher accounts are not automatically authorized instructors
+    assert @teacher.teacher?
+    refute @teacher.authorized_instructor?
+
+    # you need to be given the authorized permission
+    real_teacher = create(:teacher)
+    real_teacher.permission = UserPermission::AUTHORIZED_TEACHER
+    assert real_teacher.teacher?
+    assert real_teacher.authorized_instructor?
+
+    # or you have to be in a plc course
+    create(:plc_user_course_enrollment, user: (plc_teacher = create :teacher), plc_course: create(:plc_course))
+    assert plc_teacher.teacher?
+    assert plc_teacher.authorized_instructor?
+
+    # admins should be authorized instructors too
+    assert @admin.teacher?
+    assert @admin.authorized_instructor?
```

Comment:
do we still need admin to have this permission, now that we're going to be using code instructor?

Reply:
Ya I was wondering the same thing. I asked about it in the overview. I didn't know if since it has been working that way for authorized_teacher? if we would be worried about removing it. I guess the only people who it would impact are internal so its probably safe to remove. That would be my preference.

Reply:
ah, sorry my mistake for missing that! is it just one person (Andrea?) who would be using these admin privileges to do PLC things? if it is a small number of people then it could be quick to check with them and make sure they have an account with the new code instructor permissions available to switch over to.

also fine to remove admin in a later PR if you'd rather not bite that off right now, but I think this is ultimately worth following up on if we don't do it now.

Reply:
I don't think we use the idea of authorized teacher in PLC courses yet because there was no way to hide it from the teacher participants in the course before this point. So I think we can remove it without impacting Andrea.